00_intro/practice/tasks.md

```diff
@@ -0,0 +1,9 @@
+# **Preparation Tasks**
+
+Install your preferred Integrated development environment (IDE)
```

Comment:
I think it is better to make every word's first letter of the acronym smal or big and not to mix them.